features/FEATURE_BLE/targets/TARGET_CORDIO/TESTS/cordio_hci/driver/main.cpp

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#if !MBED_CONF_RTOS_PRESENT
```

Comment:
That should be gated by presence of BLE not RTOS

Reply:
Please review [this](https://github.com/ARMmbed/mbed-os/compare/12c84149076871aead0d1cc2e0e19170c89ee5af..e8e9c8fd6f430ebff9a92f8b9a5ed7abdd4e38c2).